freetextresponse/freetextresponse.py

```diff
@@ -760,6 +759,35 @@ def save_reponse(self, data, suffix=''):
         }
         return result
 
+    @staticmethod
+    def custom_report_format(*args, **kwargs):
+        """
+        This method return the summited answer for the give user and block.
+        """
+        user = kwargs.get('user')
+        block = kwargs.get('block')
+
+        if block and user:
+            student_item_dict = block.student_item_key(user=user)
+            submission = sub_api.get_submissions(student_item_dict, limit=1)
+            try:
+                return submission[0]['answer']
+            except IndexError:
+                pass
+
+            field = 'student_answer'
+            descriptor = modulestore().get_item(block.location, depth=1)
+            field_data_cache = FieldDataCache.cache_for_descriptor_descendents(
+                block.location.course_key,
+                user,
+                descriptor,
+                asides=XBlockAsidesConfig.possible_asides(),
+            )
+            student_data = KvsFieldData(DjangoKeyValueStore(field_data_cache))
+            if student_data.has(block, field):
+                return student_data.get(block, field)
```

Comment:
Is this returning a string value? @andrey-canon

Reply:
yes